session.go

```diff
@@ -371,7 +371,7 @@ func (s *session) idleTimeout() time.Duration {
 	if s.handshakeComplete {
 		return s.connectionParameters.GetIdleConnectionStateLifetime()
 	}
-	return protocol.InitialIdleTimeout
+	return s.config.HandshakeTimeout / 2
```

Comment:
No magic numbers… Either move the 2 to the top of the file (or server_parameters.go) or add a comment.

Reply:
This also breaks the handshake RTT test, where we test the handshake timeout. I'm not sure what to do about this, it seems like the only way to solve this is making the initial idle timeout configurable as well.